src/acl/acl.ts

```diff
@@ -142,20 +142,17 @@ export function internal_getContainerPath(resourcePath: string): string {
 }
 
 /**
- * Verify whether an ACL was found for the given Resource.
+ * Verify whether the given Resource has a resource ACL attached.
  *
- * A Resource fetched with its ACL (e.g. using [[getSolidDatasetWithAcl]]) _might_ have a resource ACL attached, but
- * we cannot be sure: it might be that none exists for this specific Resource (in which case the
- * fallback ACL applies), or the currently authenticated user (if any) might not have Control access
- * to the fetched Resource.
+ * A Resource returned using [[getSolidDatasetWithAcl]], i.e., the returned `dataset,
+ * has its resource ACL attached if its resource ACL exists and the current user
+ * has Control access to the Resource. Otherwise, the Resource will not have a resource ACL attached.
  *
- * This function verifies that the Resource's ACL is accessible.
- *
- * Please note that the Web Access Control specification is not yet finalised, and hence, this
- * function is still experimental and can change in a non-major release.
+ * _Note_: The Web Access Control specification is not yet finalised. As such, this
+ * function is still experimental and subject to change.
```

Comment:
So, I wasn't sure when you were stating that the function can change in a non-major release - whether that was exclusively only in non-major releases or in non-major releases as well as a major release.  So, I just reduced it to subject to change.

But, if it's explicitly non-major releases, let me know - because that is something to call attention to.

Reply:
Technically, as long as the major version is 0, any release can be breaking, but that's standard semver. It was specifically called out here because we know that the ACL API is particularly likely to change.

Reply:
Yeah, it could be in any release, whereas usually, breaking changes are only introduced in major releases. Thus, the comment is there to emphasise that usual expectations about breaking changes being limited to major releases do not hold for this function.

Reply:
Added back the non-major release.